### ISSUE TITLE
HDMI to DVI - Internal Display bugs when connected to External Monitor (Dell U2711)

### ISSUE THREAD

**Issue by mceykes** [bavariancake/XPS9570-macOS#54]
I am using the XPS 15's Display as a second monitor when I am at home. It is connected to a Dell U2711, which only supports 1960x1440 via Display Port and DVI. As I did not find a proper Display Port Adapter, I went for an HDMI to DVI Cable. So it is connected through the HDMI port.

Occasionally (it occurs every few minutes) the XPS Display bugs. I haven't seen it bug when I am only using the XPS. Has anyone encountered this? I updated Lilu and Whatevergreen kext. Any suggestions? Are these kext responsible for Graphics support? 

The bug looks like this: https://youtu.be/BiOAmmfDu7M

Thanks!

**Comment by mceykes** [comment]
I just noticed that the bug also occurred when there is no Monitor connected. The only thing connected is the power cable and a mouse... The amount and frequency of the bugging varies and does not happen as often when no monitor is connected.

I also tried to use the monitor when the laptop's display was shut. Then of course it worked, but then the front LED showed a color code: 7white and 2 amber flashings. This means display failure.

I tried it with an HDMI to HDMI cable and it was the same bug as with DVI. 

Any suggestions? do I need to worry about the hardware?

**Comment by bavariancake** [comment]
What happens in Windows? Does the internal display work reliably?

It doesn't look like a backlight issue, which would be something that could be fixed by WhateverGreen, so my first guess would be a bad connection from the motherboard to the internal LCD. You could try cleaning the internal connector for the LCD and re-connecting it to make sure it's seated properly.

The LCD cable is the slightly thicker one joining the WiFi/Bluetooth cables at the top left, then terminating in the center left of the image:
https://github.com/bavariancake/XPS9570-macOS/blob/master/images/3rd_antenna.jpg

Are you still having this problem, or have you resolved it?

**Comment by mceykes** [comment]
I wasn't able to try Windows because I have a Mac only setup. I tried to put in the initial SSD where Windows was still installed but it did not work as - for some reason I needed a Pin or so to start Windows. It was some kind of security mechanism. 

However, it was a connection on the Motherboard. Maybe the one to the internal LCD - however, I called the Dell support and they told me that it was probably a bad connection and that they would need to switch the entire motherboard. Luckily I just bought it and still hat warranty.

I just looked at your image, but I do not have a third antenna cable/ the LCD cable. Did you pull that one out of the monitor somehow and extended it via soldering etc?
I only have 2 connections to the bluetooth /wifistick. Does a third one make a noticeably difference?
